translation/source/faq.xml

```diff
@@ -142,4 +142,26 @@ It's best to think of ratings as "relative" figures (as opposed to "absolute" fi
 Click the lock icon next to the lichess.org address in the URL bar of your browser.
 
 Then select whether to allow or block notifications from Lichess.</string>
+  <string name="enableAutoplayForSoundsQ">Enable autoplay for sounds?</string>
+  <string name="enableAutoplayForSoundsA">Most browsers can prevent sound from playing on a freshly loaded page to protect users. Imagine if every website could immediately bombard you with audio ads.
+
+The red mute icon appears when your browser prevents lichess.org from playing a sound. Usually this restriction is lifted once you click something. On some mobile browsers, touch dragging a piece does not count as a click. In that case you must tap the board to allow sound at the start of each game.
+
+We show the red icon to alert you when this happens. Often you can explicitly allow lichess.org to play sounds. Here are instructions for doing so on recent versions of some popular browsers.</string>
+  <string name="desktop">desktop</string>
+  <string name="goToLichess">Go to lichess.org</string>
+  <string name="browsersMozillaFirefox">Press Ctrl-i on linux/windows or cmd-i on macos</string>
+  <string name="browsersMozillaFirefox1">Click the Permissions tab</string>
+  <string name="browsersMozillaFirefox2">Allow Audio and Video on lichess.org</string>
+  <string name="browsersGoogleChrome">Click the lock icon in the address bar</string>
+  <string name="browsersGoogleChrome1">Click Site Settings</string>
+  <string name="browsersGoogleChrome2">Allow Sound</string>
+  <string name="browsersSafari">Click Safari in the menu bar</string>
+  <string name="browsersSafari1">Click Settings for This Website</string>
+  <string name="browsersSafari2">Allow Auto-Play</string>
+  <string name="browsersMicrosoftEdge">Click the three dots in the top right corner</string>
+  <string name="browsersMicrosoftEdge1">Click Settings</string>
+  <string name="browsersMicrosoftEdge2">Click Cookies and Site Permissions</string>
+  <string name="browsersMicrosoftEdge3">Scroll down and click Media autoplay</string>
+  <string name="browsersMicrosoftEdge4">Add lichess.org to Allow</string>
```

Comment:
@M-DinhHoangViet Can you apply these changes? And then use these keys in `faq.scala` like this:

```scala
h3("Mozilla Firefox (", desktop(), "):"),
p(enableAutoplayForSoundsFirefox()),
h3("Google Chrome (", desktop(), "):"),
p(enableAutoplayForSoundsChrome()),
h3("Safari (", desktop(), "):"),
p(enableAutoplayForSoundsSafari()),
h3("Microsoft Edge (", desktop(), "):"),
p(enableAutoplayForSoundsMicrosoftEdge()),
```

Reply:
@M-DinhHoangViet The changes aren't reflected on this branch.

You should remove each of the individual `<li>` translations and replace with a paragraph block, so that each of the browsers has all its instructions in a single translation key.

Reply:
Apply the changes from my 2 comments above ([1](https://github.com/lichess-org/lila/pull/13922#discussion_r1383901690) and [2](https://github.com/lichess-org/lila/pull/13922#discussion_r1384933006)).

Put all the `<li>` content in a single block of text, separated by a newline, so they don't have to be translated as individual strings.

Reply:
Ok, glad it worked. Can you commit those changes to this branch? They don't show in the pull request.

Reply:
@M-DinhHoangViet That's what we want. You can commit that. Then once we apply @schlawg's suggestion about adding `nl2br`, it will appear formatted correctly.